data/sql/updates/pending_db_world/rev_1557589006688722000.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1557589006688722000');
+
+INSERT INTO `spell_group` (`id`,`spell_id`) VALUES (1052,48384),(1058,20335),(1097,14892),(1098,16176);
```

Comment:
```suggestion
DELETE FROM `spell_group` WHERE `id` IN (1052,1058,1097,1098) AND `spell_id` IN (48384,20335,14892,16176);
INSERT INTO `spell_group` (`id`,`spell_id`) VALUES 
(1052,48384),
(1058,20335),
(1097,14892),
(1098,16176);

Reply:
This would be the proper way of doing it .. :)

Reply:
But. Why?

What is the issue? 
How do I reproduce it?

Simply adding spells into random spell groups won't do nothing